### ISSUE TITLE
Cannot set attributes in an existing SQS queue

### ISSUE THREAD

**Issue by tachocarpintero** [localstack/localstack#2096]
Steps to reproduce with localstack with docker in container:
/opt/code/localstack # awslocal sqs create-queue --queue-name tacho-test
{
    "QueueUrl": "http://localstack:4576/queue/tacho-test"
}
/opt/code/localstack # awslocal sqs set-queue-attributes --queue-url http://localhost:4576/queue/tacho-test --attributes '{"FifoQueue": "true"}'

An error occurred (InvalidAttributeName) when calling the SetQueueAttributes operation: InvalidAttributeName; see the SQS docs.

Thank you in advance.

**Comment by tachocarpintero** [comment]
Also you can use this chunk of code in Java to reproduce the problem if you use com.amazonaws.aws-java-sdk-sqs library:

```
String queueUrl = sqsClient.getQueueUrl(queueName).getQueueUrl();
if (null != queueUrl) {
    sqsClient.setQueueAttributes(queueUrl, attributes);
}
```

**Comment by tachocarpintero** [comment]
FYI Workaround to solve the problem 
https://github.com/softwaremill/elasticmq/pull/340

**Comment by whummer** [comment]
Thanks for reporting @tachocarpintero . This should be fixed in #2130 - can you please give it a try with the latest version? Please report here if the problem persists. Thanks

**Comment by tachocarpintero** [comment]
Will see! I built the docker image and it went ok with my system. Thanks a lot.
